servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java

```diff
@@ -161,7 +162,7 @@ public StorageTierView getTierView(int tierAlias) {
    * @return the list of StorageTierViews
    */
   public List<StorageTierView> getTierViews() {
-    return mTierViews;
+    return Collections.unmodifiableList(mTierViews);
```

Comment:
Oh okay, but the object inside are still mutable right, so we don't get the full guarantee?